app/helpers/application_helper.rb

```diff
@@ -0,0 +1,6 @@
+module ApplicationHelper
+  def markdown(source)
+    markdown = RDiscount.new(source)
```

Comment:
Can we lock Markdown down to just the features we are using (lists and links)?

Reply:
100%